packages/protocol/contracts/governance/EpochRewards.sol

```diff
@@ -287,6 +287,15 @@ contract EpochRewards is
     return true;
   }
 
+  function setTargetVotingYield(uint256 targetVotingYield) public onlyOwner {
```

Comment:
Can we use this in `initialize`?

Reply:
i considered it, but it adds a requirement -- I didn't want to alter the current code implementation.

it does seems like it would be a benefit to the initialize to add the require statement though, so I can update